distanceTrueMean.m

```diff
@@ -0,0 +1,30 @@
+function [Dt,Gt] = distanceTrueMean(x,N,L)
```

Comment:
Maybe go for:
distanceTrue and gradientTrue.
Then everybody knows directly what is meant.